python/ray/dag/compiled_dag_node.py

```diff
@@ -1075,6 +1124,221 @@ def _get_or_compile(
         self._dag_submitter.start()
         self._dag_output_fetcher.start()
 
+    def _build_execution_schedule(self):
+        """
+        Generate an execution schedule for each actor. The schedule is a list of
+        DAGNodeOperation.
+
+        Step 1: Generate a graph based on the following rules:
+
+        #1  Divide a DAG node into three GraphNodes: READ, COMPUTE, and WRITE. Each
+            GraphNode has a DAGNodeOperation.
+        #2  Add edges between READ and COMPUTE, and between COMPUTE and WRITE, which
```

Comment:
```suggestion
        #2  Add edges from READ to COMPUTE, and from COMPUTE to WRITE, which
```
"between" is ambiguous about direction.